config/default.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/jezek/xgb"
+	"github.com/jezek/xgb/randr"
+	"github.com/jezek/xgb/xproto"
+)
+
+var (
+	AsoSxSViewerDir    = filepath.Join(os.Getenv("HOME"), ".aso_sxs_viewer")
+	AsoSxSViewerConfig = filepath.Join(AsoSxSViewerDir, "config.textproto")
+	DefaultCSSSelector = struct {
+		Selector string
+		Position int32
+	}{
+		Selector: "input",
+		Position: 7,
+	}
+	DefaultURL               = "https://mail.google.com/"
+	DefaultBrowserCount      = int32(2)
+	DefaultUseCookies        = false
+	DefaultUserDataDirPrefix = filepath.Join(AsoSxSViewerDir, "profiles")
+	DefaultRootWindowWidth   = int32(1600)
+	DefaultRootWindowHeight  = int32(900)
+	OverrideCSSSelector      = struct {
+		Selector string
+		Position int32
+	}{
+		Selector: "",
+		Position: 0,
+	}
+
+	BrowserWindowExample = `# You may use the template below to add window_overrides	
+#	window_overrides: {
+#		css_selector: {
+#			selector: ""
```

Comment:
Use sensible values in the example (maybe same as the ones you use for default), rather than empty values.